### PR TITLE
[WIP] test_containers_reports

### PR DESCRIPTION
**Pull request by gshefer**
{{pytest: cfme/tests/containers/test_containers_reports.py -v --use-provider container }}
Testing containers reports creation and verify headers (fields).

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0ce379f3289bd594b4a92d27339cf3066f859ae8:

`cfme/tests/containers/test_containers_reports.py`:
- :red_circle: Line 57:5: **N806** *variable in function should be lowercase*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit bf68b12d500a4212f532b9ee097eff654a67543e:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 852fae19208f36f2efd0e53437be6b4b44f835a0:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_containers_reports.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import pytest
+from utils import testgen
+from utils.version import current_version
+from cfme.intelligence.reports.reports import CannedSavedReport
+from collections import namedtuple
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
```
Comment (psav) [inline cfme/tests/containers/test_containers_reports.py]: The framework only supports 5.6+ now, you can drop these uncollect

File: cfme/tests/containers/test_containers_reports.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
```
Comment (psav) [inline cfme/tests/containers/test_containers_reports.py]: Suggestion, are reports really tier 1?

**Comment by dajoRH** [comment]
Lint report for commit 3255278d7b34b522d266c8bc12cc910f5a6c456d:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_containers_reports.py (hunk 2 above)
Comment (gshefer) [inline cfme/tests/containers/test_containers_reports.py]: True :)

File: cfme/tests/containers/test_containers_reports.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    Expected result:
+        * Report created successfully and has its all fields.
+    """
+    path_to_report = ['Configuration Management', 'Containers',
+                      cls.menu_name]
+    run_at = CannedSavedReport.queue_canned_report(path_to_report)
+    report = CannedSavedReport(path_to_report, run_at)
+    assert report.data.headers == cls.report_fields
+    #  TODO: Add functionality to verify that the fields of each reports are correct
```
Comment (nellyc) [inline cfme/tests/containers/test_containers_reports.py]: whats stopping us from implementing this part?

Comment (gshefer) [inline cfme/tests/containers/test_containers_reports.py]: Working on it :)

**Comment by psav** [comment]
Leaving this as WIP as per new guidelines there are comments and things to work on

**Comment by gshefer** [comment]
Moved to a new PR, more updated and more relevant: https://github.com/ManageIQ/integration_tests/pull/4467

**Pull request closed without merging**